src/utils/hooks/index.tsx

```
@@ -211,12 +211,12 @@ export const usePackInfoPrecisions = (
 
 export const useForceUpdate = (): (() => void) => {
   const [value, setValue] = useState(0);
-  return () => setValue((value) => value + 1);
+  return () => setValue(value => value + 1);
 };
 
 export const useDebounce = <T extends unknown>(
```

Biome (1.9.4)

[error] Constraining a type parameter to any or unknown is useless.

All types are subtypes of any and unknown.
Safe fix: Remove the constraint.


(lint/complexity/noUselessTypeConstraint)